src/main/webapp/app/exam/manage/suspicious-behavior/suspicious-sessions/suspicious-sessions.component.html

```diff
@@ -0,0 +1,40 @@
+<div style="overflow-x: auto; overflow-y: hidden">
```

Comment:
Why does the overflow-y need to be `hidden` here?

Reply:
it doesn't need to be, I actually only cared about the x-direction and removed it now.